core/elements/ons-switch.es6

```diff
@@ -68,16 +90,34 @@ limitations under the License.
     _compile() {
       this.classList.add('switch');
       this.innerHTML = `
-        <input type="checkbox" checked class="switch__input">
+        <input type="checkbox" class="switch__input">
         <div class="switch__toggle"></div>
       `;
       this._getCheckbox().setAttribute('name', generateId());
     }
 
     detachedCallback() {
+      this._getCheckbox().removeEventListener('change', this.onChangeListener);
     }
 
-    atachedCallback() {
+    attachedCallback() {
+      this._getCheckbox().addEventListener('change', this.onChangeListener);
+    }
+
+    onChangeListener() {
```

Comment:
Can you please prefix this method with an underscore since it's not a public API.